pkgs/servers/samba/4.x.nix

```diff
@@ -158,7 +158,7 @@ stdenv.mkDerivation rec {
     description = "The standard Windows interoperability suite of programs for Linux and Unix";
     license = licenses.gpl3;
     platforms = platforms.unix;
-    broken = stdenv.isDarwin;
+    broken = stdenv.isDarwin || enableGlusterFS;
```

Comment:
Maybe add a comment *why* this is broken when used in conjunction with `enableGlusterFS`?

Also I'd reword the commit to be `samba4: mark broken when used with enableGlusterFS option` or something.

Reply:
The build failing is the reason btw

Reply:
Alright, added a comment and changed the commit message